src/backend/gl/src/command.rs

```diff
@@ -628,3 +634,37 @@ impl command::Buffer<Resources> for CommandBuffer {
                       instances));
     }
 }
+
+impl c::CommandBuffer for CommandBuffer {
+    type SubmitInfo = SubmitInfo;
+    unsafe fn end(&mut self) -> SubmitInfo {
```

Comment:
hm this could work! Changing `SubmitInfo` to vk::CommandBuffer on vulkan would minimize code changes.